### PR TITLE
Retrieve Wx data from Env Canada

### PR DESCRIPTION
**Pull request by Kyubinhan**
* Modify Dockerfile a bit
* Add wget and autopep8 dependencies

### CODE REVIEW

File: app/env_canada.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import pytz
+import wget
+
+# pylint: disable=invalid-name, broad-except
+
+
+# Cron job schedule times are based on the timezone of the master where the job is initiated.
+tz = pytz.timezone('America/Vancouver')
+logger = logging.getLogger()
+logging.Formatter.converter = lambda *args: datetime.now(tz=tz).timetuple()
```
Comment (Sybrand) [inline app/env_canada.py:18]: Can we rather use logging.json - would prefer to if we kept the configuration in a config file.

File: Dockerfile [hunk 2]
```diff
@@ -5,23 +5,23 @@ FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8
 RUN sed -i 's/TLSv1.2/TLSv1.0/g' /etc/ssl/openssl.cnf
 
 # Install poetry
-RUN cd /tmp && \
-    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry.py && \
-    POETRY_HOME=/opt/poetry python get-poetry.py --version 1.0.8 && \
-    cd /usr/local/bin && \
-    ln -s /opt/poetry/bin/poetry && \
-    poetry config virtualenvs.create false
+RUN pip install "poetry==1.0.8"
```
Comment (Sybrand) [inline Dockerfile:8]: Poetry recommends not using pip, see:
https://python-poetry.org/docs/: "Poetry provides a custom installer that will install poetry isolated from the rest of your system by vendorizing its dependencies. This is the recommended way of installing poetry."

Comment (Kyubinhan) [reply Dockerfile:8]: Okay I will roll back the change

**Comment by Kyubinhan** [comment]
> Is there a good reason you want to deviate from the recommended poetry installation?

Oh I didn't know it was a recommended way. I like simpler stuff, that was all :)

File: Dockerfile (hunk 2 above)
Comment (Sybrand) [reply Dockerfile]: It says it adds some of it's own dependancies with pip install which may cause conflicts.

**Pull request merged**